app/serializers/student_base_serializer.rb

```diff
@@ -1,6 +1,6 @@
 class StudentBaseSerializer < BaseSerializer
 
   attributes :id, :birthday, :email,
-             :first_name, :last_name, :home_address
+             :first_name, :last_name, :home_address, :is_flagged, :is_primary
```

Comment:
To put these attributes in the `StudentIndexSerializer`, you can just define `attributes` in that serializer as above and the infrastructure will automatically merge them together as it goes up the hierarchy line!